core/start-blueos-core

```diff
@@ -133,6 +133,7 @@ SERVICES=(
     'nginx',250,"nice -18 nginx -g \"daemon off;\" -c $TOOLS_PATH/nginx/nginx.conf"
     'log_zipper',250,"nice -20 $SERVICES_PATH/log_zipper/main.py '/shortcuts/system_logs/\\\\*\\\\*/\\\\*.log' --max-age-minutes 60"
     'bag_of_holding',250,"$SERVICES_PATH/bag_of_holding/main.py"
+    'recorder',250,"blueos-recorder --recorder-path /var/logs/blueos/services/recorder"
```

Comment:
I don't think `/var/logs` is the place for recorded data -- the semantics are very different, and we should treat them differently. `/var/` is, in many cases, temporary. One quick example is that when the user deletes the system logs because they are running out of space, it shouldn't delete their mission data.

1. The service should store its data in a distinct location accessible to the user.
2. The service should (asap) add their runtime/debug logs to `/var/logs/blueos/services/recorder`

Now, what location, we may ask, and my vote is for `/usr/blueos/userdata/recorder/`.